ffi/capi/src/casemap.rs

```diff
@@ -39,6 +39,7 @@ pub mod ffi {
 
     #[diplomat::rust_link(icu::casemap::options::TitlecaseOptions, Struct)]
     #[diplomat::attr(supports = non_exhaustive_structs, rename = "TitlecaseOptions")]
+    #[diplomat::attr(kotlin, disable)] // option support (https://github.com/rust-diplomat/diplomat/issues/989)
```

Comment:
Question: Can and should this instead be more like

```
#[diplomat::attr(not(supports = option), disable)]
```

Reply:
It's about a specific subset of options, not all options. Kotlin supports _some_ types of options.

Generally I want these attributes to be about things that it makes sense for backends to not support. Partial option support is something I consider in the "backend didn't get around to it" bucket so I think per-backend suppressions are the best for that. That also indicates that kotlin may choose to add support in the future.